### ISSUE TITLE
Realtime receicer crashing

### ISSUE THREAD

**Issue by sv1** [fbieberly/ORBCOMM-receiver#4]
Hello,

I am on Debian 10 and everything is starting fine, but after a few seconds I get this: 

$ python realtime_receiver.py 
Found Rafael Micro R820T tuner

Satellites overhead: 
orbcomm fm117       : 16.7 degrees elevation
Receiving from: orbcomm fm117
Satellite frequencies: [137460000.0, 137712500.0]
Recording samples.
Create multiprocessing queue
Creating multiprocessing process
Frequency offset: -122.113502935
Bit stream offset: 78
Fill: data: B3489015957F14F39CC3 
Fill: data: D9B2B6A2EB952C9A23AD 
Fill: data: 5BC7B2637B7159843279 
Sync: code: 65A8F9 sat_id: 2F 
Fill: data: 5884E57E8DEFD2C12D2A 
Fill: data: 8C69F63A73991AA636D2 
Fill: data: 6BE5EBCAD8571C9D8981 
Fill: data: 56B412D99CE9E3148F6C 
Fill: data: 5BCFFC07B64820CBB1E1 
Fill: data: 199F6BF20EEAC995747E 
Fill: data: 414CE5337241C243DD77 
Fill: data: F0A75EA9AE310326025D 
Fill: data: 34921BE10C6FD924A500 
Fill: data: 60E43784840896BF93A3 
Fill: data: 2DAF89852278C41BA219 
Fill: data: 483E6B7369E174F69B78 
Fill: data: D1C3CE158E877B727689 
Fill: data: EFB9B7EF01F3BD2EC8CB 
Fill: data: 62D0CF10E6C6EB5B4129 
Fill: data: 8F08F3CC94BFC8389881 
Fill: data: 3CCE1007D74EF60AD937 
Fill: data: DAB7BAFDC2BB293DF78C 
Fill: data: 817A13E6CAE0EC9108F1 
Fill: data: 0B10D6CD6BBA652F4C90 
Fill: data: 4916C0AEB77D0064FA79 
Fill: data: AE5492D6697134638BD2 
Fill: data: 85DFA48C4A2BAF54FC76 
Fill: data: 036428E3816DC5C5A0D5 
Fill: data: 530472DFC6A98E8AC854 
Fill: data: 8C3CBA9958B8185B26BE 
Fill: data: 39FA4B5A83A0FD35DF61 
Fill: data: C2099BBD647969AE4AD7 
Fill: data: EAD481E2AC38D6C65D64 
Fill: data: 3815D1C5F70756E3EE14 
Fill: data: 5BF2CD2AFEBD2B5BDDFC 
Fill: data: 1B3958A56A1A7A9BE0ED 
Fill: data: 19F1D8528E1EF9701DED 
Fill: data: 5A8C1A5E354CE775C6A3 
Fill: data: 6FE4A5F1DDC8B12CADE5 
Fill: data: 567DC3683187453D728D 
Fill: data: 463E5D5FB36A15E68001 
Message: msg_packet_num: 0 msg_total_length: 1 data: 10148300000000007D 
Message: msg_packet_num: 0 msg_total_length: 2 data: 224201194940070041 
Message: msg_packet_num: 1 msg_total_length: 2 data: C2F782FA67003B17C2 
Message: msg_packet_num: 0 msg_total_length: 1 data: 33040400000000003B 
Fill: data: 3C4CEC45B4AF3662D02E 
Fill: data: C8EED0CD49CFF8EBADB1 
Fill: data: A1F27DF45A07EF620D1B 
Fill: data: 7B6EE7E16B3C9EC27C47 
Fill: data: 4836B6DE32DDD24CEBBF 
Fill: data: B3489015957F14F39CC3 
Fill: data: D9B2B6A2EB952C9A23AD 
Fill: data: 5BC7B2637B7159843279 
Sync: code: 65A8F9 sat_id: 2F 
Downlink_info: msg_packet_num: 0 msg_total_length: 3 data: B8D01150A00564003D 
Downlink_info: msg_packet_num: 1 msg_total_length: 3 data: 73D007B090101301BB 
Process Process-1:
Traceback (most recent call last):
  File "/usr/lib/python2.7/multiprocessing/process.py", line 267, in _bootstrap
    self.run()
  File "/usr/lib/python2.7/multiprocessing/process.py", line 114, in run
    self._target(*self._args, **self._kwargs)
  File "realtime_receiver.py", line 193, in process_samples
    packet_list = decoder.decode_samples_to_packets(samples, doppler)
  File "/opt/linux/ORBCOMM-receiver/realtime_decoder.py", line 376, in decode_samples_to_packets
    start_date = datetime.strptime('Jan 6 1980 00:00', '%b %d %Y %H:%M')
  File "/usr/lib/python2.7/_strptime.py", line 332, in _strptime
    (data_string, format))
ValueError: time data 'Jan 6 1980 00:00' does not match format '%b %d %Y %H:%M'
Ending async RTLSDR processing
Program exited.

Any help would be greatly appreciated!!!

**Comment by fbieberly** [comment]
Hi, Thanks for the bug report.
The only problem I can see is that strptime's %d format should be zero padded. i.e. "06" instead of "6". It works both ways for me, so I changed that and pushed it up to master. 

Could you try that out and see if it solves your problem?

**Comment by sv1** [comment]
I have already tried to zero pad the 6, but the result is the same. I am stuck!!!

**Comment by fbieberly** [comment]
Can you try a really simple test? Play around with that until you get it to work?
```
from datetime import datetime

start_date = datetime.strptime('Jan 06 1980 00:00', '%b %d %Y %H:%M')
print start_date
```

**Comment by fbieberly** [comment]
Or, a quicker/easier fix is probably to replace line #376 of the realtime_decoder.py:
```
start_date = datetime.strptime('Jan 06 1980 00:00', '%b %d %Y %H:%M')
```
with
```
start_date = datetime(year=1980, month=1, day=6, hour=0, minute=0)
```

**Comment by sv1** [comment]
> Or, a quicker/easier fix is probably to replace line #376 of the realtime_decoder.py:
> 
> ```
> start_date = datetime.strptime('Jan 06 1980 00:00', '%b %d %Y %H:%M')
> ```
> 
> with
> 
> ```
> start_date = datetime(year=1980, month=1, day=6, hour=0, minute=0)
> ```

After using the above syntax on line 376 and 451, everything seems ok!!!

Thanks!!!

**Comment by fbieberly** [comment]
I've pushed the changed to master. Closing.
